src/components/HabitForm.js

```diff
@@ -35,7 +35,7 @@ export default function HabitForm({
   const placeholderText = `neues Ziel tippen...
 `;
 
-  function updateHabit(event) {
+  function handleUpdateHabit(event) {
```

Comment:
I would rename this since you handle the update of a habit ;) e.g. updateHabit, changeHabit, handleHabitUpdate, handleHabitChange. updateHabit sounds decent to me. You could also rename habit to formHabit of formItem if this helps, so that it's clear that this about the item in the form and not the actual item that will be pushed into your habits array in App.js